src/protocols/secure_channel/PASESession.cpp

```diff
@@ -922,6 +924,8 @@ CHIP_ERROR PASESession::OnMessageReceived(ExchangeContext * exchange, const Payl
         mExchangeCtxt = nullptr;
         Clear();
         ChipLogError(SecureChannel, "Failed during PASE session setup. %s", ErrorStr(err));
+        getInstance()->pFlags.Set(getInstance()->PASEFlags::kPASESessionFailed);
```

Comment:
Do we still need this and corresponding PASEUtils change, as `CommissioningWindowManager` can deterministically close the BLE connection?

I am assuming it's not needed, as the corresponding code in `BLEManagerImpl.cpp` has been commented out. If so, let's remove it from here, and remove the commented out code.

Reply:
Removed.